src/types/point.js

```diff
@@ -29,7 +29,10 @@ export default class PointAnnotation extends Element {
     ctx.fillStyle = options.backgroundColor;
     setShadowStyle(ctx, options);
     const stroke = setBorderStyle(ctx, options);
+    // REMINDER: Sets to 0 in order to avoid that Chart.js function will perform stroking
     options.borderWidth = 0;
+    // for animation on radius, the element property must be used
+    options.radius = this.radius;
```

Comment:
even if the radius element properties was set as animation property, it was never set on element therefore never used for animation before.